### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by ONJoseph**
-add correct gitflow
-add linters
-add react using CRA
-update README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,32 +88,31 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+Feel free to check the [issues page](https://github.com/ONJoseph/math-magicians/issues).
 
-### Advanced Configuration
+## Show your support
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+Give a ⭐️ if you like this project!
 
-### Deployment
+## Acknowledgments
```
Comment (Meltrust) [inline README.md:113]: - [ ] This section on your readme is empty.  Kindly complete it according to the Microverse Readme Template.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @ONJoseph,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice package.json config✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ONJoseph) [reply README.md:113]: @Meltrust Thanks a lot for the kind review! Suggested change done.

**Review by julie-ify — APPROVED** [review]
Hi @ONJoseph, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**